### PR TITLE
build:Update to latest SDK which uses go-mod-bootstrap, updates/adds profile

### PR DESCRIPTION
**Pull request by lenny-intel**
Update go.mod to use latest SDK
Updated, modified reworked profiles
Removed sample-mqtt-msg-bus profile and replaced it with rules-engine-mqtt

### CODE REVIEW

File: res/rules-engine-mqtt/configuration.toml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    [MessageBus.PublishHost]
+        Host = "localhost"
+        Port = 1883
+        Protocol = "tcp"
+    [MessageBus.Optional]
+        # MQTT Specific options
+        # Client Identifiers
+        Username =""
+        Password =""
+        ClientId ="AppServiceSample"
```
Comment (jim-wang-intel) [inline res/rules-engine-mqtt/configuration.toml:61]: `AppServiceConfigurable`?

Comment (lenny-intel) [reply res/rules-engine-mqtt/configuration.toml:61]: Good catch meant for it to be AppServiceRulesEngine but forgot to change it...

Comment (lenny-intel) [reply res/rules-engine-mqtt/configuration.toml]: Fixed

File: res/blackbox-tests/configuration.toml [hunk 2]
```diff
@@ -95,4 +95,4 @@ Type="http"
 
 [Logging]
 EnableRemote = false
-File = "./logs/app-service-configurable-blackbox-tests.log"
+File = ""
```
Comment (rsdmike) [inline res/blackbox-tests/configuration.toml:98]: Should we not just remove this entry all together? why keep it blank?

Comment (lenny-intel) [reply res/blackbox-tests/configuration.toml:98]: So users know it exists. This is consistent with core services.

**Pull request merged**